### DISCUSSION TITLE
리프레쉬레이트 줄이는법 문의드립니다

### DISCUSSION THREAD

**Discussion by austin92p** [kimytsc/covid-rest-vaccine-macro#42] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/88754183/128955725-b989e9ec-0599-4172-8f6b-3c73d02de52e.png)

카카오톡 매크로 2일째 돌리는중인데 계속 선착순 마감에서 밀립니다 ㅠㅠ 강남지역이요 

리프레쉬레이트를 줄이는 법을 문의드립니다.

어느부분을 수정하면될까요? 임의대로 딜레이를 조금 낮췄더니 실행이되지않아 문의드립니다

**Comment by kimytsc** [comment, marked as answer]
delay의 숫자가 병원 리스트 API 호출 시작 이후 다음 호출까지의 시간텀입니다.

delay: 1000 이면 1초 이후에 목록을 다시 가져오겠다는 것인데, 너무 빠르면 일시적으로 차단되어 오히려 더 안좋아질 수 있으니 최소 500(=0.5초) 이상을 유지하시는걸 추천드립니다. 단, 500도 차단의 가능성이 있으니 주의하시길 바랍니다.

remember의 숫자는 되도록 건드리지 않으시는게 좋습니다. AZ때문에 잦은 호출이 발생하여 설정해놓은 것입니다.

**Reply by austin92p** [reply]
자세한 답변 감사드립니다..

네이버의 경우도 비슷한가요?

**Reply by kimytsc** [reply]
네, 동일합니다.
